### PR TITLE
Windows key input bug

### PR DESCRIPTION
**Pull request by sj6219**
It works abnormally when you connect to the window server and type shift + left arrow.
It's been modified.

### CODE REVIEW

**Comment by rustdesk** [comment]
@asur4s please help review

File: src/server/input_service.rs [hunk 1]
```diff
@@ -648,11 +648,12 @@ fn map_keyboard_mode(evt: &KeyEvent) {
     let (click_capslock, click_numlock) = sync_status(evt);
 
     #[cfg(windows)]
-    {
+    let click_numlock = {
     	crate::platform::windows::try_change_desktop();
 
         let code = evt.chr();
         let key = rdev::get_win_key(code, 0);
+        //let key = rdev::windows::keycodes::key_from_code(code);
```
Comment (chiehw) [inline src/server/input_service.rs]: Remove it.

File: src/server/input_service.rs [hunk 2]
```diff
@@ -648,11 +648,12 @@ fn map_keyboard_mode(evt: &KeyEvent) {
     let (click_capslock, click_numlock) = sync_status(evt);
 
     #[cfg(windows)]
-    {
+    let click_numlock = {
```
Comment (chiehw) [inline src/server/input_service.rs:644]: It looks good, it would be even better if it could be written in the sync_status function

**Review by chiehw — CHANGES_REQUESTED** [review]

**Comment by sj6219** [comment]
I modified it.

**Review by chiehw — APPROVED** [review]

**Comment by asur4s** [comment]
LGTM

**Comment by rustdesk** [comment]
Thanks @sj6219  @asur4s

**Pull request merged**